Rohit/PostgresAssignment/sql/topper.ts

```diff
@@ -0,0 +1,29 @@
+const topclient = require("../database/dbconnect");
+
+exports.get_topper_by_classid_and_subjectid = async (
+    c_id: number,
+    sub_id: number
+  ) => {
+    await topclient.query("set search_path to myschool");
+    return await topclient.query(
+      `select s_id,s_name,S.marks from (select * from result where clas_id=${c_id} and subjectid=${sub_id} order by marks desc) as S,students where S.studentid=s_id limit 1`
+    );
+  };
+
+  exports.get_topten_students = async (c_id: number) => {
+    await topclient.query("set search_path to myschool");
+    return await topclient.query(`select s.s_id , s.s_name , a.total_marks
+            from myschool.students s
```

Comment:
can you check https://sequelize.org/master/ for query. Its a ORM,s it give function to do this kind of things. Try it out no need to change in this assignment